wordpress-mobile/WordPress-iOS
# Login Epilogue: adjust table width for iPad

Ref #13413 

This updates the Login Epilogue's site list table width based on the size class and device orientation.
- For the Regular x Regular size class:
  - In portrait, the table is 66% of the view width.
  - In landscape, the table is 50% of the view width.
- For all other size classes, the table is 100% of the view width.

In addition, the Done button is now always the same width as the table.

To test:

---
- Login on an iPad.
- Verify the table and button width are sized properly in portrait and landscape.

| ![ipad_portrait](https://user-images.githubusercontent.com/1816888/78945388-6ad3fd80-7a7d-11ea-9d51-df21f4f918af.png) | ![ipad_landscape](https://user-images.githubusercontent.com/1816888/78945400-7293a200-7a7d-11ea-9430-54c69f717894.png) |
|--------|-------|

---
- Login on an iPhone, or multitask on an iPad and change the view size.
- Verify the table and button width are sized properly.

![iphone](https://user-images.githubusercontent.com/1816888/78945412-77f0ec80-7a7d-11ea-9c19-ff5e66b2dbbb.png)

---
PR submission checklist:
